src/compiler/utilities.ts

```diff
@@ -2615,11 +2620,19 @@ namespace ts {
         });
     }
 
-    /** Get the type annotaion for the value parameter. */
-    export function getSetAccessorTypeAnnotationNode(accessor: SetAccessorDeclaration): TypeNode {
+    /** Get the type annotation for the value parameter. */
+    export function getSetAccessorTypeAnnotationNode(accessor: SetAccessorDeclaration, includeJSDocType?: boolean): TypeNode {

```

Comment:
Shouldn't `getAnnotatedAccessorType` and `isGetAccessorWithAnnotatedSetAccessor` call this with `includeJSDocType`? Would it ever hurt to include that?